plugins/search-react/src/extensions.tsx

```diff
@@ -105,9 +105,13 @@ const SearchResultListItemExtension = ({
   }, [rank, result, noTrack, analytics]);
 
   return (
-    <Box role="button" tabIndex={0} onClickCapture={handleClickCapture}>
+    <ListItem
+      divider
+      alignItems="flex-start"
```

Comment:
I'm not sure if we should force all result items to align items this way; maybe the style can be moved inside the result item components?

Reply:
I had this debate with myself, and since the pattern was there already to align all list items to flex-start I didn't see a reason to not bring this over. Can you think of use cases where one would like to align search result list items differently?

Reply:
https://github.com/backstage/backstage/pull/15873/commits/b3ef4d8f73640b5f661f38b58a6f789dcb739249